src/util.h

```diff
@@ -971,7 +971,7 @@ void SetConstructorFunction(v8::Isolate* isolate,
 // Simple RAII class to spin up a v8::Isolate instance.
 class RAIIIsolate {
  public:
-  RAIIIsolate();
+  RAIIIsolate(const SnapshotData* data = nullptr);
```

Comment:
```suggestion
  explicit RAIIIsolate(const SnapshotData* data = nullptr);
```
to fix the linter error